### PR TITLE
added input fields

### PR DESCRIPTION
**Pull request by mastershailesh1234**
tried to solve issue #3

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,9 +13,20 @@
...
+    <h1>Welcome To Confession Page</h1>
     <div id="root"></div>
+    <div class="container inputarea">
+      <fieldset>
+      <legend>Enter details for confessions</legend>
+      <h1>Name of the person you want to confess</h1>
+      <input class="card" type="text" name="name" placeholder="Name" autocomplete="off">
+      <h1>Write your confession here</h1>
+      <textarea class="card" name="name" maxlength="200" placeholder="Enter confess in 200 max words........."></textarea>
+      <button type="submit" name="button">Submit</button>
```
Comment (iharshit009) [inline index.html:25]: Can we beautify, this button?

File: index.html [hunk 2]
```diff
@@ -13,9 +13,20 @@
...
 <body>
+    <h1>Welcome To Confession Page</h1>
     <div id="root"></div>
+    <div class="container inputarea">
+      <fieldset>
+      <legend>Enter details for confessions</legend>
+      <h1>Name of the person you want to confess</h1>
+      <input class="card" type="text" name="name" placeholder="Name" autocomplete="off">
+      <h1>Write your confession here</h1>
+      <textarea class="card" name="name" maxlength="200" placeholder="Enter confess in 200 max words........."></textarea>
```
Comment (iharshit009) [inline index.html:24]: Input text
Make sure textarea and input name field, have the same border margin, they look different.

**Review by iharshit009 — COMMENTED** [review]
Hey @mastershailesh1234, you have done impressive work, please make these minor changes so that I can merge the PR.

File: index.html (hunk 1 above)
Comment (mastershailesh1234) [reply index.html:25]: yaaa i forgot to beautify it. i will do that.

File: index.html (hunk 2 above)
Comment (mastershailesh1234) [reply index.html:24]: okayy okayy iwill correct it.

**Comment by mastershailesh1234** [comment]
i have added some change. see if its ryt now.

File: index.html [hunk 3]
```diff
@@ -13,16 +13,16 @@
...
     <div class="container inputarea">
       <fieldset>
       <legend>Enter details for confessions</legend>
       <h1>Name of the person you want to confess</h1>
       <input class="card" type="text" name="name" placeholder="Name" autocomplete="off">
       <h1>Write your confession here</h1>
-      <textarea class="card" name="name" maxlength="200" placeholder="Enter confess in 200 max words........."></textarea>
-      <button type="submit" name="button">Submit</button>
+      <textarea class="card" name="name" maxlength="200" placeholder="Enter confession in 200 max words........."></textarea>
+      <button type="submit" name="button">🤐Confess😅</button>
```
Comment (iharshit009) [inline index.html:25]: Great one😅😅

**Comment by iharshit009** [comment]
@mastershailesh1234 Great job man
Keep Learning!

**Pull request merged**